sci-mathematics/netgen/netgen-5.3.1.ebuild

```diff
@@ -0,0 +1,103 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit autotools eutils flag-o-matic multilib versionator
+
+MY_PN=${PN}-mesher
+MY_PV=$(get_version_component_range 1-2)
+DESCRIPTION="NETGEN is an automatic 3d tetrahedral mesh generator"
+HOMEPAGE="http://www.hpfem.jku.at/netgen/"
+SRC_URI="mirror://sourceforge/project/${MY_PN}/${MY_PN}/${MY_PV}/${P}.tar.gz"
+
+SLOT="0"
+LICENSE="LGPL-2.1"
+KEYWORDS="~amd64 ~x86"
+IUSE="-ffmpeg jpeg -mpi opencascade"
+
+DEPEND="
+	dev-tcltk/tix
+	dev-tcltk/togl:1.7
+	virtual/opengl
+	x11-libs/libXmu
+	opencascade? ( sci-libs/opencascade:* )
+	ffmpeg? ( media-video/ffmpeg )
+	jpeg? ( virtual/jpeg:0= )
+	mpi? ( virtual/mpi ( || ( sci-libs/parmetis <sci-libs/metis-5.0 ) ) sci-libs/hdf5[mpi] ) "
```

Comment:
There is one set of ```()``` too much. Also, is hdf5 only required for mpi support? Not for sequential?